pkgs/tools/networking/imapsync/default.nix

```
@@ -67,7 +67,7 @@ stdenv.mkDerivation rec {
     mainProgram = "imapsync";
     homepage = "https://imapsync.lamiral.info/";
     license = licenses.nlpl;
-    maintainers = with maintainers; [ pSub ];
+    maintainers = with maintainers; [ pSub motiejus ];
```

This is not nixfmt

```suggestion
    maintainers = with maintainers; [
      pSub
      motiejus
    ];
```

Please check result with formatter